code/src/java/pcgen/gui2/csheet/CharacterSheetPanel.java

```diff
@@ -103,17 +103,13 @@ public CharacterSheetPanel()
 
 		// KAW TODO rewrite to use StatusWorker and PCGenTask for better progress display
 
-		refreshService = Executors.newSingleThreadExecutor(new ThreadFactory()
+		refreshService = Executors.newSingleThreadExecutor(r ->
 		{
-			@Override
-			public Thread newThread(Runnable r)
-			{
-				Thread thread = new Thread(r, "Character-Sheet-Refresher-Thread");
-				thread.setDaemon(true);
-				thread.setPriority(Thread.NORM_PRIORITY);
-				return thread;
-			}
-		});
+            Thread thread = new Thread(r, "Character-Sheet-Refresher-Thread");
```

Comment:
I suspect this is just a setting in my IDE which needs to get turned on again. Will find it and fix.